src/Language/PureScript/Docs/RenderedCode/RenderType.hs

```diff
@@ -154,12 +154,6 @@ insertPlaceholders :: RenderTypeOptions -> Type -> Type
 insertPlaceholders opts =
   everywhereOnTypesTopDown convertForAlls . everywhereOnTypes (convert opts)
 
-dePrim :: Type -> Type
-dePrim ty@(TypeConstructor (Qualified _ name))
-  | ty == tyBoolean || ty == tyNumber || ty == tyString =
-    TypeConstructor $ Qualified Nothing name
-dePrim other = other
```

Comment:
I think I wrote this code originally, and this module was mostly copied from `Language.PureScript.Pretty.Types` as it was at that time - of course this was the behaviour we wanted in `prettyPrintType`, but not what we wanted here, although I failed to realise that at the time.